projects/fsastorefrontlib/src/core/my-account/facade/claim.service.ts

```diff
@@ -183,6 +183,21 @@ export class ClaimService {
       .unsubscribe();
   }
 
+  changeClaim(claim: Claim) {
+    this.userIdService
+      .getUserId()
+      .pipe(take(1))
+      .subscribe(occUserId => {
```

Comment:
We should avoid subscribing in the service when we can. You could call `this.userIdService.getUserId()` in `submit` when call `combineLatest` and extend this method with additional parameter `userId`.

Reply:
getUserId observable moved to combineLatest in component